test/suites/pch.bash

```diff
@@ -55,7 +55,9 @@ SUITE_pch() {
 
     pch_suite_common
     if $COMPILER_TYPE_CLANG; then
-        pch_suite_clang
+        if ! [ $COMPILER_BIN = "gcc" ] || $RUN_WIN_XFAIL; then
```

Comment:
Can `$COMPILER_BIN` ever be `gcc` here since `$COMPILER_TYPE_CLANG` is true?

Reply:
Yes. That's the case on MSYS2/Clang{32,64}. You run gcc, but it's in fact clang.